ostack/deploy.sh

```diff
@@ -28,11 +37,10 @@ printf '%(%Y%m%d-%H:%M:%S)T\n'
 cd $DPL
 echo "cwd=$PWD"
 
-echo "PXD=$PXD"
+export PXD=PXD011124
```

Comment:
This is basically the _only_ input the user would have to provide and the value of `$PXD` should come from a field in the portal.